src/js/popup.js

```diff
@@ -284,6 +284,7 @@ function send_error(message) {
         $("#report_cancel").prop("disabled", false);
         $("#report_success").toggleClass("hidden", true);
         closeOverlay();
+        delete POPUP_DATA.errorText;
```

Comment:
This doesn't resolve the issue of error text not being cleared after submitting a report successfully and then closing the entire popup before the report UI autocloses. Since the `setTimeout` callback may never happen, we might want to send the "removeErrorText" message earlier, from outside the callback.